buildSrc/src/main/groovy/airbyte-bulk-connector.gradle

```diff
@@ -68,6 +68,7 @@ class AirbyteBulkConnectorExtension {
         // Use a temporary configuration to find the Micronaut version
         // in the transitive dependencies of the Bulk CDK base.
         def tempConfiguration = project.configurations.detachedConfiguration()
+        // NOTE: I think this is failing: ("Could not get unknown property 'first' for project of type org.gradle.api.Project.")
```

Comment:
I was not able to debug this and I'll probably need assistance to do so. Commenting out `cdk = '...'` worked but I think there's something else going on.

My best analysis is that in places like like 73, we are asking for `.first` as if this is a collection or list object, where it is actually already a singular `Project` object. That might be wrong, and I'm not at all clear why no one else would be running into this besides myself. Could be a version conflict or or issue on my local workspace, although I tried all the obvious things like aligning JDK and Gradle version... 🤔

Reply:
We could simply choose to not apply formatting on gradle-groovy files. I believe there's an effort underway led by @jdpgrailsdev to move to gradle-kotlin.